src/shared_utilities/drivers.jl

```diff
@@ -41,15 +41,21 @@ Container for holding prescribed atmospheric drivers and other
 information needed for computing turbulent surface fluxes when
 driving land models in standalone mode.
 
-The arguments labeled with "function of time" are typically either of type
-`Function` or type `Spline1D` from the `Dierckx.jl` package.
-
 Since not all models require co2 concentration, the default for that
 is `nothing`.
```

Comment:
this was something I forgot to update when I changed the default. we now just use the current atmos co2 concentration as the default. 

"The default CO2 concentration is a constant as a function of time, equal to 4.2e-4 mol/mol."

Reply:
Added